packages/core/scripts/admin-proposals/setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
nit: maybe we should call this and setupFork something more specific to impersonating accounts? Otherwise, it's unclear what this script is and in what cases it needs to be run.